src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj

```diff
@@ -6,13 +6,12 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <Version>3.3.0</Version>
+    <Version>3.4.0</Version>
```

Comment:
Can't see any changes to the SDK itself, please regenerate the code if needed. 
Else, please confirm you wish to publish a Nuget with no new changes in the SDK.

Reply:
Yeah, forgot that the new generated code wasn't auto-generated on commit to azure-rest-api-specs.  Added